perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/GroupsManagerBlImpl.java

```diff
@@ -3353,41 +3347,19 @@ private void synchronizeSubGroupsMembers(PerunSession sess, Group baseGroup, Ext
 			groupMemberExtsource.setValue(baseMemberExtsource.getValue());
 			getPerunBl().getAttributesManagerBl().setAttribute(sess, group, groupMemberExtsource);
 
-			synchronizeMembersAndSaveResult(sess, group);
-		}
+			Attribute isThisLightweightSynchro = getPerunBl().getAttributesManagerBl().getAttribute(sess, baseGroup, GroupsManager.GROUPLIGHTWEIGHTSYNCHRONIZATION_ATTRNAME);
```

Comment:
If this is really meant to get the attribute for baseGroup, I believe you can put it before the 'for' loop.

Reply:
fixed